package/MDAnalysis/core/groups.py

```diff
@@ -734,6 +739,8 @@ def center(self, weights, pbc=None, compound='group'):
 
         comp = compound.lower()
         if comp == 'group':
+            if unwrap:
+                coords = atoms.unwrap(compound=comp, reference=None, inplace=False)
             if pbc:
                 coords = atoms.pack_into_box(inplace=False)
```

Comment:
Having both `unwrap=True` and `pbc=True` doesn't make sense if `compound='group'`.
IMHO this should throw either an exception or a warning stating the precedence of operations.
I'm in favor of throwing an exception.